tracex/patient_journey_generator/views.py

```diff
@@ -0,0 +1,60 @@
+"""This file contains the views for the patient journey generator app."""
+from django.urls import reverse_lazy
+from django.views import generic
+from extraction.forms import GenerationForm
+from extraction.logic.orchestrator import Orchestrator, ExtractionConfiguration
+from tracex.logic import constants
+from . import patient_journey_generator
+
+IS_TEST = False  # Controls the presentation mode of the pipeline, set to False if you want to run the pipeline
+
+
+class JourneyGeneratorOverviewView(generic.TemplateView):
+    """View for the landing page of the patient journey generator."""
+
+    template_name = "journey_generator_overview.html"
+
+
+class JourneyGenerationView(generic.CreateView):
+    """View for generating a patient journey."""
+
+    form_class = GenerationForm
+    template_name = "journey_generation.html"
+    success_url = reverse_lazy("processing")
+
+    def get_context_data(self, **kwargs):
+        """Generate a patient journey and save it in the cache."""
+        context = super().get_context_data(**kwargs)
+        orchestrator = Orchestrator()
+
+        if IS_TEST:
+            with open(
+                str(constants.input_path / "journey_synth_covid_1.txt"), "r"
+            ) as file:
+                journey = file.read()
+                configuration = ExtractionConfiguration(patient_journey=journey)
+                orchestrator.set_configuration(configuration)
+        else:
+            # This automatically updates the configuration with the generated patient journey
```

Comment:
Comment not really necessary here as it only explains what the code already says